tests/unit/test_bids_utils.py

```diff
@@ -81,3 +86,66 @@ def test_bidsmap_scans_run_plus():
     scans = [("1", "run+"), ("2", "run+"), ("3", "run+")]
 
     assert bidsmap_scans(scans) == [("1", "run-01"), ("2", "run-02"), ("3", "run-03")]
+
+
+def test_bidsify_dicom_headers(mocker):
+    """Test bidsify_dicom_headers without ProtocolName field"""
+    dataset = mocker.MagicMock()
+    dataset.__contains__.return_value = False
+
+    dcmread = mocker.patch("pydicom.dcmread")
+    dcmread.return_value = dataset
+
+    bidsify_dicom_headers("filename", "foo")
+
+    assert dataset.data_element.called == False
```

Comment:
3. Using the `dataset` mock we created previously, we can check to see if the method `data_element` has ever been called on that mock. In this particular test we don't expect it to be called, because the function returns early if `"ProtocolName"` is not in `dataset`